great_expectations/datasource/new_datasource.py

```diff
@@ -39,7 +39,7 @@ def __init__(
 
         Args:
             name: the name for the datasource
-            execution_engine (ClassConfig): the type of compute engine to produce
+            execution_engine: compute engine configuration with keys 'module_name' and 'class_name'.
```

Comment:
I didn't change the type here but I am updating the docstring. This actually expects a dictionary and not a `ClassConfig`.